docs/querying/examples.md

```diff
@@ -95,3 +95,22 @@ Assuming this metric contains one time series per running instance, you could
 count the number of running instances per application like this:
 
     count by (app) (instance_cpu_time_ns)
+
+If we are exploring some metrics for their labels, to e.g. be able to aggregate
+over some of them, we could use the following:
+
+    limitk(10, app_foo_metric_bar)
+
+Alternatively, if we wanted the returned timeseries to be more evenly sampled,
```

Comment:
I really like these new additional explanations and examples, but they are a bit out of place in this introductory PromQL examples page that otherwise only mentions a handful of the most popular language features. Explaining these details under https://prometheus.io/docs/prometheus/latest/querying/operators/#aggregation-operators would be better IMO.

Reply:
Ah okies, will move.

Reply:
Moved+improved docu at 80e4463.

Reply:
Also added that _poor-man's-normality-check_ to `limit.test` at fe21d5a696a3dc0b369080bdb106ca18d1ffd36e.